statsmodels/tsa/vector_ar/tests/test_var.py

```diff
@@ -15,6 +15,18 @@
 import numpy as np
 import pytest
 
+have_matplotlib = False
+try:
```

Comment:
This block is pretty common, may merit putting in something `like sm.tools._testing`, especially since it has now grown past 2-3 lines.

Reply:
(not necessarily for this PR)